dashboard/src/modules/components/DatePickerComponent/DatePicker.test.js

```diff
@@ -0,0 +1,30 @@
+import { Provider } from "react-redux";
+import store from "store/store";
+import { MOCK_DATA } from "utils/mockData";
+import App from "../../../App";
+const { render, screen, fireEvent } = require("@testing-library/react");
+const AppWrapper = () => {
+  return (
+    <Provider store={store}>
+      <App />
+    </Provider>
+  );
+};
+jest.mock("utils/api", () => {
+  return {
+    get: () => ({
+      data: MOCK_DATA,
+    }),
+  };
+});
+test("data is filtered based on date range selected from date picker", async () => {
+  render(<AppWrapper />);
+  await screen.findByText("dhcp1");
+  const datePickerInput = screen.getAllByPlaceholderText(/yyyy-mm-dd/i);
+  fireEvent.change(datePickerInput[0], { target: { value: "2022-02-16" } });
```

Comment:
Shouldn't we be checking that the UI has updated from `YYYY-MM-DD` to `2022-02-16`?

Also, can the code here look at the values in the React-store?  I think those would be good things to check, as well.